docs/reference/sso.md

```diff
@@ -0,0 +1,522 @@
+
+#### Add a secret
+Sometimes called an application password, a client secret is a string value your app can use in place of a certificate to identity itself.
+
+Client secrets are considered less secure than certificate credentials. Application developers sometimes use client secrets during local app development because of their ease of use. However, you should use certificate credentials for any of your applications that are running in production.
+
+Steps:
+1. In the Azure portal, in App registrations, select your application.
+2. Select Certificates & secrets > Client secrets > New client secret.
+3. Add a description for your client secret.
+4. Select an expiration for the secret or specify a custom lifetime.
+    1. Client secret lifetime is limited to two years (24 months) or less. You can't specify a custom lifetime longer than 24 months.
+    2. Microsoft recommends that you set an expiration value of less than 12 months.
+5. Select Add.
+6. Record the secret's value for use in your client application code. This secret value is never displayed again after you leave this page.
+
+#### Add a redirect URI
+A redirect URI is the location where the Microsoft identity platform redirects a user's client and sends security tokens after authentication.
+
+You add and modify redirect URIs for your registered applications by configuring their platform settings.
+
+Enter https://lakefs-cloud.us.auth0.com/login/callback or https://lakefs-cloud.eu.auth0.com/login/callback (depends on your organization data location) as your redirect URI.
+
+Settings for each application type, including redirect URIs, are configured in Platform configurations in the Azure portal. Some platforms, like Web and Single-page applications, require you to manually specify a redirect URI. For other platforms, like mobile and desktop, you can select from redirect URIs generated for you when you configure their other settings.
+
+Steps:
+1. In the Azure portal, in App registrations, select your application.
+2. Under Manage, select Authentication.
+3. Under Platform configurations, select Add a platform.
+4. Under Configure platforms, select the web option.
+5. Select Configure to complete the platform configuration.
+
+Once you finish registering lakeFS Cloud with Azure AD, save the **Application (Client) ID**, **Application Secret Value** and send this to Treeverse's team to finish the integration.
+
+## Enterprise
+
+lakeFS Enterprise provides a secondary service, running side-by-side with lakeFS which handles the authentication, this service is called Fluffy.
+Once you've onboarded to [lakeFS Enterprise](../enterprise.md) and have access to the authentication service (Fluffy), you can configure one of the supported authentication methods below.
+lakeFS Enterprise leverage Helm's capabilities to run the sidecar with it's configuration, on top of the specific-authentication method, please review the [Helm configuration](#helm).
```

Comment:
This sentence doesn't make sense. I don't understand what it's saying so I can't rewrite it. Is it two sentences that have run together? 

```suggestion
lakeFS Enterprise uses Helm's capabilities to run the sidecar with its configuration, on top of the specific-authentication method, please review the [Helm configuration](#helm).
```

Reply:
rephrased it, please let me know if it's still doesn't make sense